File: Truncate-Sentence.cpp
class Solution {
public:
    string truncateSentence(string s, int k) {
        string ans = "";
        int count = 0;
        for (int i = 0; i < s.size(); i++) {
            if (s[i] == ' ') {
                count++;
                if (count == k) {
                    ans = s.substr(0, i);
                    return ans;
                }
            }
        }
        return s;
    }
};

File: Truncate-Sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function (s, k) {
    let ans = "";
    let count = 0;
    for (let i = 0; i < s.length; i++) {
        if (s[i] === ' ') {
            count++;
            if (count === k) {
                ans = s.slice(0, i);
                return ans;
            }
        }
    }
    return s;
};